### PR TITLE
new favicons

### LINKED ISSUE #21: Update Favicon

### PR DESCRIPTION
**Pull request by usefulthink**
closes #21

### CODE REVIEW

File: contents/manifest.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"template": "partials/manifest.json.njk", "filename": "manifest.json"}
```
Comment (usefulthink) [inline contents/manifest.json]: thats a bit wild, but json-files in the content-tree are interpreted as pages..

**Comment by lukaszklis** [comment]
@verpixelt good catch! :D

**Comment by usefulthink** [comment]
read somewhere that despite the filename the actual image should be larger (something something hidpi), but I have no real idea about how the content should look in that case (neither do I even know how it is used... assets were supplied by @Kriesse as they are.

**Comment by lukaszklis** [comment]
We will address the mstile issue if necessary later on.

**Pull request merged**